truffle/src/main/java/org/jruby/truffle/language/backtrace/BacktraceFormatter.java

```diff
@@ -128,6 +128,10 @@ public void printBacktrace(RubyContext context, DynamicObject exception, Backtra
     private String formatInLine(List<Activation> activations, DynamicObject exception) {
         final StringBuilder builder = new StringBuilder();
 
+        if (activations.size() == 0) {
```

Comment:
There are differing opinions on this, but it might be clearer to call `activations.isEmpty()`.